packages/jest-haste-map/src/fastpath.js

```diff
@@ -48,590 +48,589 @@ const util = require('util');
 // fastpath is only useful on older versions of nodejs.
 if (parseInt(process.versions.node.split('.')[0], 10) >= 5) {
   module.exports = Object.assign({replace: () => {}}, path);
-  return;
```

Comment:
had to change indentation for the whole file because of this `return` outside of function :/